### PR TITLE
Overlay: deprecate option 'copyRootClassesToWrapper' and add option to hide deprecate warning

### PR DESCRIPTION
**Pull request by Zedwag**

### CODE REVIEW

File: js/ui/diagram/ui.diagram.floating_panel.js [hunk 1]
```diff
@@ -125,6 +125,7 @@ class DiagramFloatingPanel extends DiagramPanel {
             position: this._getPopupPosition(),
             showCloseButton: true,
             copyRootClassesToWrapper: true,
+            hideCopyClassesDeprecationWarning: true,
```
Comment (ksercs) [inline js/ui/diagram/ui.diagram.floating_panel.js]: Let's name it `_ignoreCopyRootClassesToWrapperDeprecation` by analogy with `_ignoreFunctionValueDeprecation`

```suggestion
            _ignore: true,
```

File: js/ui/overlay/ui.overlay.js [hunk 2]
```diff
@@ -240,6 +243,14 @@ const Overlay = Widget.inherit({
         });
     },
 
+    ctor: function(element, options) {
+        this.callBase(element, options);
+
+        if(options && options.copyRootClassesToWrapper && !options.hideCopyClassesDeprecationWarning) {
+            errors.log('W0001', this.NAME, 'copyRootClassesToWrapper', '21.2', 'Use the "wrapperAttr" option instead');
```
Comment (ksercs) [inline js/ui/overlay/ui.overlay.js:235]: why `W001`? We should add a new warning.

Comment (ksercs) [inline js/ui/overlay/ui.overlay.js:235]: What about runtime change?

File: js/ui/diagram/ui.diagram.floating_panel.js (hunk 1 above)
Comment (Zedwag) [reply js/ui/diagram/ui.diagram.floating_panel.js]: ok

File: js/ui/overlay/ui.overlay.js (hunk 2 above)
Comment (Zedwag) [reply js/ui/overlay/ui.overlay.js:235]: afaik `W001` is warning for all deprecated options

Comment (Zedwag) [reply js/ui/overlay/ui.overlay.js:235]: earlier we desided not to proccess runtime `copyRootClassesToWrapper`

**Review by ksercs — APPROVED** [review]

**Review by alexanderPolosatov — APPROVED** [review]

**Review by alexander-kotov-dx — APPROVED** [review]

**Review by DokaRus — APPROVED** [review]

**Pull request merged**